<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>file_organiser</class>
 <widget class="QMainWindow" name="file_organiser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>291</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>487</width>
    <height>291</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>File Organiser</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">color:black;</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>25</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <property name="verticalSpacing">
          <number>35</number>
         </property>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QCheckBox" name="cb_mp3">
             <property name="text">
              <string>.mp3</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cb_wav">
             <property name="text">
              <string>.wav</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cb_jpg">
             <property name="text">
              <string>.jpg</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cb_png">
             <property name="text">
              <string>.png</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QCheckBox" name="cb_pdf">
             <property name="text">
              <string>.pdf</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cb_txt">
             <property name="text">
              <string>.txt</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cb_mp4">
             <property name="text">
              <string>.mp4</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cb_mov">
             <property name="text">
              <string>.mov</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPushButton" name="pb_browse">
             <property name="text">
              <string>Browse</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_organise">
             <property name="text">
              <string>Organise</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>487</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>cb_mp3</tabstop>
  <tabstop>cb_wav</tabstop>
  <tabstop>cb_jpg</tabstop>
  <tabstop>cb_png</tabstop>
  <tabstop>cb_pdf</tabstop>
  <tabstop>cb_txt</tabstop>
  <tabstop>cb_mp4</tabstop>
  <tabstop>cb_mov</tabstop>
  <tabstop>pb_browse</tabstop>
  <tabstop>pb_organise</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
